packages/loader/container-loader/README.md

```diff
@@ -68,9 +68,13 @@ Usually container is returned when state of container (and data stores) is rehyd
 
 ### Closure
 
-Container can be closed directly by host by calling `Container.close()`. Once closed, container terminates connection to ordering service, and any local changes (former or future) do not propagate to storage.
+Container can be closed directly by host by calling either `Container.close()` or `Container.dispose()`. If the container is expected to be used upon closure, use the `close()` API. Otherwise, use the `dispose()` API. The differences between these methods are detailed in the sections below.
 
-Container can also be closed by runtime itself as result of some critical error. Critical errors can be internal (like violation in op ordering invariants), or external (file was deleted). Please see [Error Handling](#Error-handling) for more details
+#### `Container.close()`
+
+Once closed, container terminates connection to ordering service, and any local changes (former or future) do not propagate to storage.
```

Comment:
I believe we do not allow "future" changes in closed state (mostly indirectly, through "readonly" events and expectations that components will honor them).